core/server/worker/src/main/java/alluxio/worker/netty/DataServerUfsBlockReadHandler.java

```diff
@@ -69,6 +68,8 @@ public void close() {
           LOG.warn("Failed to close block reader for block {}.", mId);
         }
       }
+      // Add this back when the commit block logic is moved to Netty
+      // mWorker.cleanupSession(mSessionId);
```

Comment:
A temporary work around could be to expose if an error occurred and only clean up then, but I think this is unnecessary since we plan to address it in this feature branch. Also, cleaning up in all cases will ensure we obey the invariant of completing the entire operation before closing.